Core/bin/getenv.sh

```diff
@@ -35,20 +35,33 @@ resolve_path() {
 
 mango_keystore_properties() {
   if [ -z "$MA_KEYSTORE" ]; then
-    keystore="$(get_prop "ssl.keystore.location" "keystore.p12")"
+    keystoreMessage="keystore variable populated with"
+    keystoreDefault="keystore.p12"
+    keystoreProperty="ssl.keystore.location"
+    keystore="$(get_prop $keystoreProperty $keystoreDefault)"
+    print_value_source_message "$keystore" "$keystoreDefault" "$keystoreMessage" "$keystoreProperty"
     MA_KEYSTORE="$(resolve_path "$mango_paths_data" "$keystore")"
+    print_ma_keystore_value_source_message "$MA_KEYSTORE" "$keystore" "MA_KEYSTORE environment variable populated with"
```

Comment:
In situations where the default value for the path to the keystore.p12 file is correct, this script will print a warning message about using the default value because the expected value was not found. We shouldn't print a warning message about using the default value if the default value points to a file that exists.